chaoscenter/graphql/server/pkg/environment/handler/handler_test.go

```diff
@@ -0,0 +1,170 @@
+package handler_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/graph/model"
+	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/authorization"
+	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb/environments"
+	"github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/environment/handler"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type MockEnvironmentOperator struct {
```

Comment:
How about mocking the Mongo Operator instead of mocking the Environment Operator? This allows you to write test cases that are flexible to future changes in the Environment Operator. How do you think?

```golang
import (
  dbMocks "github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb/mocks"
  dbOperationsEnvironment "github.com/litmuschaos/litmus/chaoscenter/graphql/server/pkg/database/mongodb/environments"
)

mongodbMockOperator := new(dbMocks.MongoOperator)
environmentOperator := dbOperationsEnvironment.NewEnvironmentOperator(mongodbMockOperator)
```


reference: https://github.com/litmuschaos/litmus/blob/master/chaoscenter/graphql/server/pkg/choas_experiment_run/service_test.go

Reply:
I think I could go with Environment Operator since I already updated the handler to consider an environment Mockoperator

Reply:
okay, thanks